Twig/Extension/SonataAdminExtension.php

```diff
@@ -437,6 +437,11 @@ protected function getTemplate(
         try {
             $template = $environment->loadTemplate($templateName);
         } catch (\Twig_Error_Loader $e) {
+            @trigger_error(
+                'Relying on default template loading on field template '.
+                'loading exception is deprecated and will be removed in 4.0',
```

Comment:
Maybe you should specify what will happen since 4.0 and prepare the commented code of the thrown exception.